README.md

```diff
@@ -117,5 +116,13 @@ The document must be wrapped in `<pdf></pdf>` tags
             - `src="String"` Source of the image to draw relative to the template file
             - `scale="Number"` Size of image relative to its original
             - `fit="Boolean"` Fit image within the width and height without chaning its aspect ratio
+            - `alt="String"` Add alt text to an image to improve accessibility
         * `<a>` Create a web link for the contained text.
             - `href="String"` The link to go to when clicked. If no `href` is specified an href is created by adding `https://` to the beginning of the text
+
+## Version 2.0.0
```

Comment:
Like Sam mentioned, I think we just delete the version notes here and keep that in the GitHub releases.